internal/js_printer/js_printer.go

```diff
@@ -2634,28 +2628,6 @@ func (p *printer) printBlock(loc logger.Loc, stmts []js_ast.Stmt) {
 	p.print("}")
 }
 
-/* This needs to be kept in sync with `printBlock`'s implementation. */
-func (p *printer) printReactComponentFnBody(loc logger.Loc, stmts []js_ast.Stmt, refreshSignatureVarName string) {
-	p.addSourceMapping(loc)
-	p.print("{")
-	p.printNewline()
-
-	p.options.Indent++
-	p.printIndent()
-	p.print(refreshSignatureVarName + "();")
```

Comment:
Happy to see this go, but something like this may be needed to get de-duplicated hook names into signatures.